templates/hooks/certificate_authority/sdk_create_post_request.go.tpl

```diff
@@ -0,0 +1,3 @@
+    if err != nil && strings.HasPrefix(err.Error(), "RequestInProgressException") {
+            return nil, ackrequeue.Needed(err)
```

Comment:
Requeue causes an exponential backoff retry, ideally you want to requeue after 5/10seconds (or what ever value you think it's appropriate)